src/modules/Dropdown/Dropdown.js

```diff
@@ -1167,6 +1169,7 @@ export default class Dropdown extends Component {
       onClick: this.handleItemClick,
       selected: selectedIndex === i,
       ...opt,
+      key: getKeyOrValue(opt.key, opt.value),
```

Comment:
You are correct, however, this is a semver fail on my part.  I should have incremented the minor (since we're pre 1.0).

I do not think we should ship this feature moving forward.  In `1.0` we will not automatically generate keys when using shorthand objects or elements.  Users will have to pass a `key` manually.  If we add this now, it will encourage users to rely on `value` being used as the `key` (at least in the Dropdown only).

If we feel very strongly about fixing this regression, I am willing to ship this as a _patch_ right now to take care of current users as long as we immediately follow it up with a `breaking(): ...` PR that removes it for `0.69.0`.